handlers/admin.py

```
@@ -0,0 +1,355 @@
+        )
+        
+        # Уведомляем пользователя
+        from main import bot
+        try:
+            action_text = "пополнен" if amount > 0 else "списан"
+            await bot.send_message(
+                chat_id=user_id,
+                text=f"💰 **Изменение баланса**\n\n"
+                     f"Ваш баланс {action_text} на {abs(amount)} ₽ администратором."
+            )
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)